### ISSUE TITLE
Question: helm3 test no parallel option anymore

### ISSUE THREAD

**Issue by GianniGiglio** [helm/helm#10921]
Question is it still possible to run helm tests in parallel.
Helm2 had the option --parallel but that seems to be gone in helm2

**Comment by bacongobbler** [comment]
https://github.com/helm/helm/pull/4144#issuecomment-587094629

Duplicate of #7763. See https://github.com/helm/helm/pull/8946 and the current status of the HIP at https://github.com/helm/community/pull/165#issuecomment-1079402279.
